src/NServiceBus.Transport.Msmq.AcceptanceTests/TimeToBeReceived/When_message_with_expired_ttbr_header_is_received.cs

```diff
@@ -31,7 +33,8 @@ public async Task Message_should_not_be_processed()
                         ctx.WasSent = true;
                     })
                 )
-                .Run(TimeSpan.FromSeconds(5));
```

Comment:
that didn't actually wait 5 seconds. Since there was no Done condition, the test would complete immediately, causing it to fail sometimes if the sent message hadn't been picked up yet.